routes/bookRoutes.js

```diff
@@ -0,0 +1,63 @@
+import express from 'express';
+import { Book } from '../models/bookModel';
+
+const router = express.Router();
+
+// Helper function for handling errors
+const handleErrors = (res, error) => {
```

Comment:
nice to create a error function, this could be put in a map called middleware and maybe also be its own component:)